packages/runtime/test/lib/testUtils.ts

```diff
@@ -508,12 +508,14 @@ export async function waitForRecipientToReceiveNotification(
 }
 
 /**
- * finds how many attributes will be there in addition to the desired ones
- * Assumes that
+ * finds how many attributes will be there from previous tests in addition to the desired ones
  */
 export async function syncAndGetBaselineNumberOfAttributes(sender: TestRuntimeServices, filter: GetAttributesRequest): Promise<number> {
     await sleep(1000);
```

Comment:
I didn't reviewed the last tests PR, can we get rid of the sleep in any way?

sleeps in tests are bad b/c
1. they bloat the test time (could be that they wait to long)
2. the don't wait long enough and everything explodes

Reply:
I think it's possible but querying the outcoming resp. incoming requests to decide which ones are in the status "recipient accepted but sender has not synced" also feels awkward

Reply:
I have replaced it, but the success of this check run required re-running the mongodb test

Reply:
means it becomes flaky b/c you removed the sleep?

Reply:
Tests have always been flaky - the test run that ended up "approving" the merge into main didn't require re-running, then later without any changes it has failed again. Also I re-ran Test #356 from before the change and it also succeeded.

Reply:
I mean - sleeps can never be the cure for flakyness. If it's still flaky you should go over the tests again.

Reply:
those specific tests ("check the messageDVO for the sender after acceptance") in the (Complex)ReadAttributeDVO tests look fine to me - I'll ask someone to have a look at it